redfish-core/lib/led.hpp

```diff
@@ -548,11 +576,23 @@ inline void
 
             for (auto& endpoint : *endpoints)
             {
-                auto callback = [aResp, endpoint, locationIndicatorActive](
-                                    const std::string& serviceName) {
+                bool ledOn = locationIndicatorActive;
+                if (!ledState.empty())
+                {
+                    if (boost::ends_with(endpoint, "_blink"))
+                    {
+                        ledOn = ledState == "Blinking" ? true : false;
+                    }
+                    else
+                    {
+                        ledOn = ledState == "Lit" ? true : false;
```

Comment:
Typically don't use the Ternary operator in bmcweb

Reply:
Has been updated to upstream